cli/tools/installer.rs

```diff
@@ -381,20 +382,22 @@ async fn install_global(
   ));
 
   let entry_text = install_flags_global.module_url.as_str();
-  let req = super::registry::AddRmPackageReq::parse(entry_text);
-
-  // found a package requirement but missing the prefix
-  if let Ok(Err(package_req)) = req {
-    if jsr_resolver.req_to_nv(&package_req).await.is_some() {
-      bail!(
-        "{entry_text} is missing a prefix. Did you mean `{}`?",
-        crate::colors::yellow(format!("deno install -g jsr:{package_req}"))
-      );
-    } else if npm_resolver.req_to_nv(&package_req).await.is_some() {
-      bail!(
-        "{entry_text} is missing a prefix. Did you mean `{}`?",
-        crate::colors::yellow(format!("deno install -g npm:{package_req}"))
-      );
+  if !cli_options.initial_cwd().join(entry_text).exists() {
```

Comment:
Isn't this going to hit the file system twice when the file exists? That seems like a redundant file operation to me

Reply:
It's fine. There won't be any noticable perf impact from one file system read.

Reply:
Alright